fullCalendar/fullCalendar.d.ts

```diff
@@ -318,7 +318,7 @@ interface JQuery {
     /**
      * Moves the calendar to an arbitrary date.
      */
-    fullCalendar(method: 'gotoDate', date: Date | string): void;
+    fullCalendar(method: 'gotoDate', moment.Moment | Date | string): void;
```

Comment:
That changing looks like good.
But this line is occurred syntax error. Because the variable name is missing.